lms/djangoapps/learner_dashboard/config/waffle.py

```diff
@@ -37,20 +37,3 @@
     'learner_dashboard.enable_masters_program_tab_view',
     __name__,
 )
-
-# .. toggle_name: learner_dashboard.enable_b2c_subscriptions
-# .. toggle_implementation: WaffleFlag
-# .. toggle_default: False
-# .. toggle_description: Waffle flag to enable new B2C Subscriptions Program data.
-#    This flag is used to decide whether we need to enable program subscription related properties in program listing
-#    and detail pages.
-# .. toggle_use_cases: temporary
-# .. toggle_creation_date: 2023-04-13
-# .. toggle_target_removal_date: 2023-07-01
-# .. toggle_warning: When the flag is ON, the new B2C Subscriptions Program data will be enabled in program listing
-#    and detail pages.
-# .. toggle_tickets: PON-79
-ENABLE_B2C_SUBSCRIPTIONS = WaffleFlag(
-    'learner_dashboard.enable_b2c_subscriptions',
-    __name__,
-)
```

Comment:
Has this Waffle flag been removed? If not, is there a ticket or task in the backlog to remove this flag?

Reply:
Not yet, it's on my todo list but I added to the original ticket. I might remove today since we're moving over to RV on Monday